compendium/DeclarativeServices/src/manager/ConfigurationNotifier.hpp

```diff
@@ -96,14 +97,13 @@ namespace cppmicroservices
 
             void UnregisterListener(std::string const& pid, const cppmicroservices::ListenerTokenId token) noexcept;
 
-            bool AnyListenersForPid(std::string const& pid) noexcept;
+            bool AnyListenersForPid(std::string const& pid, cppmicroservices::AnyMap const& properties) noexcept;
```

Comment:
Should this function be "const"? I don't see any instance data being modified in the implementation.

Reply:
It doesn't compile if I add the const. I think it's the listenersMap.lock function call.